src/Pages/SearchPages/DataRequestPages/DataRequest.js

```diff
@@ -15,37 +15,41 @@ export default class DataRequest extends Component {
         var RAD_hash_link = "/search/" + transaction.RAD_bytes_hash;
         var block_link = "/search/" + transaction.block;
         return (
-            <Table style={{"marginBottom": "0px"}}>
+            <Table responsive>
```

Comment:
Icon padding is also too large in data requests tab

![image](https://github.com/user-attachments/assets/36857528-ebeb-4b12-a5db-9fab11d2d735)